### PR TITLE
Title link and highlight viewing 2

### PR DESCRIPTION
**Pull request by BrianYing**

### CODE REVIEW

File: tests/webview/ui/test_content.py [hunk 1]
```diff
@@ -799,3 +799,26 @@ def test_toc_button_labelled_books(webview_base_url, selenium, page_id):
...
+    # THEN click the title
+    content.get_left_nav_book_title.click()
+
+    # AND get and click the Contents button
+    content.header_nav.click_contents_button()
+
+    # AND find the on viewing title and get the color
+    chapter_4 = content.header_nav.table_of_contents.chapters[-1]
+    page_6_color = chapter_4.pages[-1].color()
+    assert page_6_color == right_color
```
Comment (m1yag1) [inline tests/webview/ui/test_content.py]: Where is the THEN statements. 

Remember all asserts should be under THEN.

File: pages/webview/content.py [hunk 2]
```diff
@@ -31,6 +32,10 @@ class Content(Page):
                           ' button[type="submit"][data-l10n-id="textbook-view-btn-get-this-book"]')
     )
     _ncy_locator = (By.CLASS_NAME, 'not-converted-yet')
+    _book_info_title_locator = (By.CSS_SELECTOR, '.booksContaining > div > span')
```
Comment (m1yag1) [inline pages/webview/content.py]: Books containing is specific to a different page. If you need access to `booksContaining` then you must use `pages/content_page.py` which represents a single page all by itself w/o a book.

File: pages/webview/content.py [hunk 3]
```diff
@@ -31,6 +32,10 @@ class Content(Page):
...
+    _go_to_book_link_locator = (By.CSS_SELECTOR, 'li:nth-child(3) > div > a')
+    _left_nav_book_title_locator = (By.CSS_SELECTOR, 'div.booksContaining > ul > li:nth-child(1) '
+                                                     '> div > a > b')
```
Comment (m1yag1) [inline pages/webview/content.py]: In `pages/content_page.py` there is a Region called Book that represents a book in the book list. There is also a property on the page called `ContentPage.books_containing.books_list` where these elements should probably live.

**Comment by m1yag1** [comment]
What issues does this PR link to?

**Comment by BrianYing** [comment]
#227

**Pull request merged**